apps/devu-api/utils/llm-chat/tools/look-things-up-online.ts

```
@@ -0,0 +1,213 @@
+  }
+  const htmlString = await response.text().catch(() => 'Error: failed to fetch search result')
+  if (htmlString.startsWith('Error:')) {
+    return htmlString
+  }
+
+  const { document } = parseHTML(htmlString)
+  const turndownService = new TurndownService()
+
+  let zeroClickInfoTable: HTMLTableElement | null = null
+  for (const td of Array.from(document.querySelectorAll('td'))) {
+    if (td.textContent && td.textContent.toLowerCase().includes('zero-click info:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)